src/pages/Crunchyroll/beta.ts

```diff
@@ -89,7 +89,7 @@ export const beta: pageInterface = {
     return Boolean(j.$('.erc-watch-episode-layout').length);
   },
   isOverviewPage(url) {
-    return Boolean(j.$('.erc-series-hero').length);
```

Comment:
We specially did not want to use the url for the check, because crunchyroll does strange things with its urls sometimes. They used to translate the url paths, which currently seems like not beeing the case anymore. But we wanted to be safe. But apparently this backfired. But you are right, now it is probably stable enough to use the url.

Still this change will only work for the US version of the website as the other languages have a locale prefix in the url. 
https://www.crunchyroll.com/de/series/G4PH0WJQZ/fatestrange-fake

Reply:
Ah, I was wondering why this DOM selector and not a simple url check. Let me see if I can find another selector to hook up to.

Reply:
Done, replaced with `.erc-season-episode-list`.

Reply:
The German page you link works now too.